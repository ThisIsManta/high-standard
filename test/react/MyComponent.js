import React from 'react'
import ReactDOM from 'react-dom/client'

export default function MyComponent() {
  return <div></div>
}

ReactDOM.createRoot(document.getElementById('react')).render(<MyComponent />)