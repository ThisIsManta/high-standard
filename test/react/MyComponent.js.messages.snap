error 2:8 [levitate/import-convention] Unexpected the default import.
error 5:10 [@stylistic/jsx-self-closing-comp] Empty components are self-closing
error 5:10 [react/self-closing-comp] Empty components are self-closing
error 8:10 [levitate/import-convention] Expected "createRoot" to be imported directly.
error 8:21 [no-undef] 'document' is not defined.
error 8:78 [@stylistic/eol-last] Newline required at end of file but not found.